service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/C2DocumentBundle.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fpl.model.common;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class C2DocumentBundle {
+    private final DocumentReference document;
+    private final String description;
+    private final String uploadedDateTime;
```

Comment:
I wonder if we shouldn't give the date to CCD to display. How the dates are displayed by default? Design not happy with it?

Reply:
We talked about this, needs further exploration.